translations/es.json

```diff
@@ -0,0 +1,742 @@
+					"title": "Synth {{currencyKey}} pausado",
+					"subtitle": "Por favor consulte el Discord de Synthetix para más detalles."
+				},
+				"circuit-breaker": {
+					"title": "Cortacircuito activado para {{currencyKey}}",
+					"subtitle": "Reajuste en progreso"
+				},
+				"emergency": {
+					"title": "Apagado de emergencia activado para {{currencyKey}}",
+					"subtitle": "Reajuste en progreso"
+				}
+			},
+			"tooltip": {
+				"close": "cierre",
+				"high": "máximo",
+				"low": "mínimo",
+				"open": "apertura",
+				"price": "precio"
+			},
+			"no-data": "Sin datos"
+		},
+		"summary-info": {
+			"gas-price-gwei": "precio del gas (GWEI)",
+			"fee": "comisión (fee)",
+			"fee-cost": "coste de la comisión",
+			"button": {
+				"approve": "aprobar",
+				"approving": "aprobando",
+				"c-ratio-too-low": "el c-ratio es demasiado bajo",
+				"fee-reclaim-period": "periodo de reclamación de la comisión",
+				"insufficient-balance": "balance insuficiente",
+				"enter-amount": "introduzca la cantidad",
+				"select-synth": "seleccionar synth",
+				"select-token": "seleccionar token",
+				"submit-order": "enviar orden",
+				"submitting-order": "enviando orden...",
+				"submitting-approval": "enviando approval...",
+				"approve-balancer": "aprobar contrato de balancer",
+				"add-more-collateral": "añadir más colateral"
+			}
+		},
+		"onboard": {
+			"message": "Tu wallet no tiene Synths. Compra <0>{{currencyKey}}</0> para hacer trading con Synths:",
+			"l2-message": "Tu wallet no tiene Synths. Crea <0>{{currencyKey}}</0> haciendo staking para hacer trading con Synths:",
+			"mint-button": "Ir a Staking"
+		},
+		"connect-wallet-card": {
+			"message": "Conecta tu wallet para empezar a hacer trading"
+		},
+		"footer-card": {
+			"market-closure": {
+				"button-label": "mercado cerrado",
+				"reasons": {
+					"frozen": {
+						"message": "El synth {{currencyKey}} ha sido congelado - reajuste en progreso."
+					},
+					"market-closure": {
+						"message": "El mercado {{currencyKey}} ha sido cerrado - reajuste en progreso."
+					},
+					"system-upgrade": {
+						"message": "Synth {{currencyKey}} pausado. Por favor consulte el Discord de Synthetix para más detalles."
+					},
+					"circuit-breaker": {
+						"message": "Cortacircuito activado para {{currencyKey}} - reajuste en progreso."
+					},
+					"emergency": {
+						"message": "Apagado de emergencia activado para {{currencyKey}} - reajuste en progreso."
+					},
+					"both-synths-suspended": {
+						"message": "Los Synths seleccionados están actualmente pausados - reajuste en progreso."
+					},
+					"after-hours": {
+						"message": "El mercado de valores está cerrado. Haz trading durante las horas de cierre en Balancer",
+						"button-label": "Abrir la pool de Balancer"
+					}
+				}
+			},
+			"get-l2-gas": {
+				"message": "No hay suficiente wETH de L2 para las tarifas (fees) de la transacción. Adquiere un poco vía Optimism Bridge:"
```

Comment:
I suggest to remove ``(fees)`` and call it ``tarifas de gas`` to avoid confusion between ``tarifas`` (transaction gas costs in wETH) and ``comisión (fee)`` which is the seperate Kwenta fee of currently ~.4% for the trade a user makes, which are two different things.